docs/DeveloperGuide.md

```diff
@@ -1217,30 +1219,213 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+2. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
+### Adding a patient
+
+1. Adding a patient
+
+    1. Prerequisites: There exist no patient with NRIC `T1258979J` in the database.
+
+    2. Test case (Valid parameters): `add_p n\Bunny Cai Hong ic\T1258979J p\83425673`<br>
```

Comment:
your siambu gf?

Reply:
you like?

![image](https://github.com/AY2324S1-CS2103T-T08-1/tp/assets/24840938/8bac4d63-3b37-46e9-8926-3388c859be1f)